docs/pages/abide.md

```diff
@@ -61,7 +61,7 @@ These input types create a text field: `text`, `date`, `datetime`, `datetime-loc
     <div class="grid-x grid-margin-x">
       <div class="cell large-6">
         <label>URL Pattern, not required, but throws error if it doesn't match the Regular Expression for a valid URL.
-          <input type="text" placeholder="https://foundation.zurb.com" pattern="url">
+          <input type="text" placeholder="https://www.get.foundation" pattern="url">
```

Comment:
Do we need the www at the beginning? I would prefer the short version. Easier to type and shorter inn general.

Reply:
the official domain has www because from my experience when people see just get.foundation, they are not aware that its a domain. the www makes them realize that. There are redirect to www if you do not include it.